.env

```diff
@@ -16,6 +16,7 @@
 ###> symfony/framework-bundle ###
 APP_ENV=dev
 APP_SECRET=394ab0d3123ec3b35f78b29fb6b56eed
+TRUSTED_PROXIES=127.0.0.1,REMOTE_ADDR
```

Comment:
This adds the REMOTE_ADDR always as a trusted proxy? Does that not defy the meaning of "trusted"?